locations/spiders/screwfix_gb.py

```diff
@@ -1,14 +1,27 @@
+import html
+import json
+
 from scrapy.linkextractors import LinkExtractor
 from scrapy.spiders import CrawlSpider, Rule
 
-from locations.structured_data_spider import StructuredDataSpider
+from locations.linked_data_parser import LinkedDataParser
 
 
-class ScrewfixGBSpider(CrawlSpider, StructuredDataSpider):
+class ScrewfixGBSpider(CrawlSpider):
     name = "screwfix_gb"
     item_attributes = {"brand": "Screwfix", "brand_wikidata": "Q7439115"}
     allowed_domains = ["www.screwfix.com"]
-    download_delay = 0.1
+    download_delay = 1
```

Comment:
```suggestion
```

We don't need to define the default, this way we can change it later without having to change spiders.